### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by Honey14**
Referring slack message : https://simpledotorg.slack.com/archives/CFKMFQCKB/p1608550819128500?thread_ts=1608550724.128400&cid=CFKMFQCKB

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # CHANGELOG
 
-## On Demo
-### Features
-- Add BP Passport sheet
```
Comment (msasikanth) [inline CHANGELOG.md:5]: We haven't moved it to `On Demo` or `Next Release`, is this expected?

**Review by msasikanth — CHANGES_REQUESTED** [review]
Left one comment PTAL.

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:5]: NVM, forgot to read the message 😅

Comment (Honey14) [reply CHANGELOG.md:5]: Since this is in progress and this will not be visible to nurses yet. Hence in the next release.

**Review by msasikanth — APPROVED** [review]

**Review by msasikanth — APPROVED** [review]

**Pull request merged**